documentation/aks-upgrade.md

```diff
@@ -105,3 +105,14 @@ az aks nodepool get-upgrades --resource-group <ResourceGroup> --cluster-name <Cl
 1. Follow the same manual process to upgrade the production cluster
     - Test with the welcome app: https://www.teacherservices.cloud/
     - Raise PR
+## Monitoring upgrade
```

Comment:
This is useful during the upgrade. Can it be added to "Upgrade process"? Alongside:
Continuously monitor it during the upgrade and check for downtime. Example: